jekyll/_cci2/insights-glossary.md

```diff
@@ -0,0 +1,129 @@
+| Runs                  | The count of executions in a selected time window.                                                                                                                                                                                                                                                                     |
+| Total Duration        | The sum of execution times for all workflows in a project.                                                                                                                                                                                                                                                             |
+| P95 Duration          | The 95th percentile of execution times for an entity in a selected time window (i.e. 95% of runs completed in the same or less amount of this time). <br/> _The 95th percentile is a standard measure used to interpret performance data. It provides a measure of max value when outlier or transient values are excluded._ |
+| P50 (median) Duration | The median execution time. <br/> _Medians are a better measure of central tendency than arithmetic means because they are more resilient to_ skewness _in datasets_.                                                                                                                                                         |
+| Total Credits         | The sum of credits consumed during execution.                                                                                                                                                                                                                                                                          |
+| Success Rate          | The percentage of runs that completed successfully, calculated by `100 x (Successful Runs / All Runs)`                                                                                                                                                                                                                   |
+{: class="table table-striped"}
+
+## Organization-level metrics
+
+![]({{ site.baseurl }}/assets/img/docs/insights-org-metrics.png)
+
+Organization-level metrics allow you to analyze your organization’s performance.
+
+| Term                    | Definition     |
+|-------------------------|----------------|
+| Workflow Runs | The count of all workflows executions for an organization for the selected projects in the selected time frame. |
+| Total Workflow Duration | The total execution time of all workflows runs for an organization within the selected projects and time frame. |
+| Total Credits | The total credits consumed across all selected projects in an organization. |
+| Overall Success Rate | The percentage of runs that completed successfully across all runs in the selected projects and workflows. (calculated by `100 x (All Successful Runs/ All Runs)`) |
+{: class="table table-striped"}
+
+## Workflow Metrics
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 41. You can fix by inserting `{: #workflow-metrics }` to the next line